README.md

```diff
@@ -1,9 +1,99 @@
 
-We are currently working towards the first major release of MewsFeed.
+MewsFeed is a micro-blogging platform built with [Holochain](https://holochain.org).
 
-[Collaborate on GitHub](https://github.com/orgs/GeekGene/projects/1) and [join our Discord](https://discord.gg/D3BykUZumM)!
+Visit [mewsfeed.social](https://mewsfeed.social) to be notified of our release!
+
+[Collaborate on GitHub](https://github.com/orgs/GeekGene/projects/1), [join our Discord](https://discord.gg/D3BykUZumM), and donate on [OpenCollective](https://opencollective.com/mewsfeed)!
+
+## How to use MewsFeed
+
+MewsFeed is built with the distributed app development framework Holochain. Practically, this means that users either host themselves on their own devices or access a web version through [the Holo hosting infrastructure](https://holo.host).
+
+Here is how to decide which version to use:
+
+#### 1. You are a Tinkerer 🛠️
+You like to experiment with the tools you use and enjoy having access to the data you use.
+
+💻 Install MewsFeed locally, either as a Desktop App with Kangaroo or in the Holochain Launcher.
+
+This way you can analyze your data, connect it to other tools, and tinker around.
+
+#### 2. You are a Creator 🧑‍💻
+You like to publish good posts and interact with your audience wherever you are.
+
+📲 Sign up for an account on [app.mewsfeed.social](https://app.mewsfeed.social).
+
+This way you can use it from any device through the browser. Alternatively, install MewsFeed locally and you'll be able to create content even when you're offline.
+
+#### 3. You are an Anon 👤
+You want to stay anonymous, shitpost, and interact with interesting accounts.
+
+💻 Install MewsFeed locally, either as a Desktop App with Kangaroo or in the Holochain Launcher.
+
+This way you have access to your posts even when you don't have internet.
+
+#### 4. You are a Lurker 👀
+You mostly just want to follow others for their content and stay informed of what's happening.
+
+📲 Sign up for an account on [app.mewsfeed.social](https://app.mewsfeed.social) and use it from any device through the browser.
+
+Alternatively, if you only browse content from your laptop or desktop, install it locally.
+
+Either way, you'll be able to follow people and read their stuff.
+
+#### 5. You are a casual User 😎
+You like following cool accounts, interacting with your friends, and having interesting conversations.
+
+💻 Install MewsFeed locally, either as a Desktop App with Kangaroo or in the Holochain Launcher.
+
+This way you can access your favorite content and send replies to your friends even when you're offline. Alternatively, sign up on [app.mewsfeed.social](https://app.mewsfeed.social) and use it from any device through the browser.
+
+### Install Options
+
+#### As Desktop App
+Download the latest release of the Desktop App here: [MewsFeed Kangaroo](https://github.com/GeekGene/mewsfeed-kangaroo/releases/tag/v0.10.1).
+
+Once installed, you'll be greeted by the Explore page.
```

Comment:
I don't think the additional usage instructions are necessary. If we want to keep them, maybe move into a separate section "Using the hApp" since they're not really related to installation.